CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - t3c-generate fix for combining remapconfig and cachekeyconfig parameters for MakeRemapDotConfig call.
 - [#6780](https://github.com/apache/trafficcontrol/issues/6780) Fixed t3c to use secondary parents when there are no primary parents available.
 - Correction where using the placeholder `__HOSTNAME__` in "unknown" files (others than the defaults ones), was being replaced by the full FQDN instead of the shot hostname.
+- [#7007](https://github.com/apache/trafficcontrol/issues/7007) Fixed t3c timeout and retries not working.
```

Comment:
You put this entry in the 7.0.0 section, but no 7.x RC exists that contains this change

Reply:
Ah, missed the section header. Fixed.